torchaudio/models/wavernn.py

```diff
@@ -324,3 +334,37 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def _wavernn(arch: str, pretrained: bool, progress: bool, **kwargs: Any) -> WaveRNN:
```

Comment:
Looking at the code, there is no point of having this as a helper function. Performing this directly in `wavernn_10k_epochs_8bits_ljspeech` function is more readable.

Note that torchvision's resnet adopts this pattern because it has many factory functions that only differ in configuration, and [Don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is a common theme in software engineering.

Reply:
I think this should be resolved with the new interface.
Thanks.